src/Sarif/Visitors/SarifVersionOneToCurrentVisitor.cs

```diff
@@ -519,21 +565,34 @@ internal Result CreateResult(ResultVersionOne v1Result)
             {
                 result = new Result
                 {
-                    AnalysisTarget = CreateFileLocation(v1Result.Locations?[0]?.AnalysisTarget),
```

Comment:
Bug: The fix here is not to remove `AnalysisTarget`. It's to populate it _only if it differs from the result file_.

Reply:
Oh! I see you did that later. ok.

Reply:
Right, it's handled conditionally down after the result creation